### DISCUSSION TITLE
Wondering if there will be a playground for CUE

### DISCUSSION THREAD

**Discussion by ryuheechul** [cuelang/cue#517] [General] [answered]
Hi all 👋, 
I just discovered CUE and read some docs and about to start using it.
I think CUE's potential is quite great and a playground might help more adoptions.
Since I couldn't find any issues or discussions about playground topic, just wanted to throw it out there to see what people's view on it.

**Comment by sahuguet** [comment, marked as answer]
https://cuelang.org/play/#cue@export@cue

**Reply by myitcv** [reply]
I'll also note that https://github.com/cuelang/cuelang.org/issues/110 is about making that more prominent (although we likely need to optimise the site first)

**Reply by ryuheechul** [reply]
Thanks @sahuguet ! I guess I somehow couldn’t find it before 😅.

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/517.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
